03/index.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
     <link rel="stylesheet" href="./styles/global.css">
-    <link rel="stylesheet" media="(min-width: 599px) and (max-width:1299px)" href="./styles/mobile.css">
+    <link rel="stylesheet" media="(max-width: 599px) and (max-width:1299px)" href="./styles/mobile.css">
```

Comment:
Jacku, dlaczego masz 2 `max-width`? Jak byś to odczytał?

> Wczytaj plik mobile.css tylko wtedy gdy maksymalna szerokość to 599 oraz gdy maksymalna szerokość to 1299px.

W naszym przypadku interesuje nas tylko pierwsza cześć wiec powinno być: ` <link rel="stylesheet" media="(max-width: 599px)" href="./styles/mobile.css">`